framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -252,7 +252,7 @@ public int runTests() throws Exception {
       for (DrillTestJdbc test : verificationFailures) {
         LOG.info(test.getInputFile());
         LOG.info("Query: \n" + test.getQuery());
-        LOG.info(test.getException().getMessage());
+        LOG.info(test.getException() == null? "":test.getException().getMessage());
```

Comment:
(Better question) We need to ensure that test.getException() does not return null when there is a verification failure, so the cause is known. Is there a case where `test.getException() == null` is true?